src/solver/conv_direct_naive_conv_bwd.cpp

```diff
@@ -102,6 +81,40 @@ ConvSolution ConvDirectNaiveConvBwd::GetSolution(const ConvolutionContext& ctx)
     int c_per_group = c / group;
     int k_per_group = k / group;
 
+    int block_size = 256;
+    int grid_size  = 1;
+    if(ctx.IsLayoutDefault())
+    {
+        grid_size = n * c;
+    }
+    else if(ctx.IsLayoutNHWC())
+    {
+        if(ctx.Is2d())
+            grid_size = group * n * hi;
+        else
+            grid_size = group * n * di;
+    }
+    else
+        MIOPEN_THROW("Unsupported layout");
+
+    KernelInfo kernel;
+
+    kernel.kernel_file = ConvDirectNaiveConvKernelFile(ctx);
+    kernel.kernel_name = ConvDirectNaiveConvKernelName(ctx);
+    kernel.g_wk.clear();
+
+    kernel.g_wk.push_back(grid_size * block_size);
```

Comment:
[Notice] If there is possibility that `grid_size * block_size` exceeds INT_MAX, please consider using `size_t`. `g_wk` and `l_wk` in `KernelInfo` are vectors of `size_t`.

Reply:
OK, let me change this

Reply:
?